azure/durable_functions/models/DurableOrchestrationContext.py

```diff
@@ -214,7 +214,8 @@ def call_activity_with_retry(self,
 
     def call_http(self, method: str, uri: str, content: Optional[str] = None,
                   headers: Optional[Dict[str, str]] = None,
-                  token_source: TokenSource = None) -> TaskBase:
+                  token_source: TokenSource = None,
+                  is_raw_str: bool = False) -> TaskBase:
```

Comment:
Inside this method, there's comments explaining each parameter. Please add `is_raw_string` to that set. Thanks!

Reply:
Thanks! just added.